### PR TITLE
content(data tiering): expand

### PR DESCRIPTION
**Pull request by charislam**
Add more details and context to the data tiering docs

### CODE REVIEW

File: _partials/_cloud-data-tiering-no-untier.md [hunk 1]
```diff
@@ -0,0 +1,5 @@
+<highlight type="warning">
+Data tiering currently works one way. Data can be moved from standard storage to
+object storage, but not in reverse. This capability, `untier_chunk`, is planned
+for a future beta release.
```
Comment (Loquacity) [inline _partials/_cloud-data-tiering-no-untier.md:4]: Few thoughts on this wording. The "works one way" thing is confusing as a standalone sentence. Can we combine the first and second sentences so it seems a little less confusing? Also, I'm really not OK with forward facing statements like the final sentence, can we be a little more circumspect with it? "could be introduced in a future release" maybe? 🤔

File: cloud/data-tiering/index.md [hunk 2]
```diff
@@ -16,8 +17,8 @@ Save on storage costs by tiering data to a low-cost object-storage layer.
 Timescale Cloud includes a low-cost object-storage layer built on Amazon S3.
 This allows you to tier your hypertable data across different storage layers to
 get the best price performance. You can use primary storage for data that
-requires quick access, and low-cost object storage for historical data. You can
-query all your data with standard SQL, no matter where it's stored.
+requires quick access, and low-cost object storage for historical data. No
+matter where your data is stored, you can still query it with standard SQL.
```
Comment (Loquacity) [inline cloud/data-tiering/index.md:21]: "no matter" might not translate well.

```suggestion
requires quick access, and low-cost object storage for historical data. 
Regardless of where your data is stored, you can query it with standard SQL.
```

File: cloud/data-tiering/index.md [hunk 3]
```diff
@@ -35,11 +36,40 @@ With data tiering, you get:
...
+    when it's distributed across different storage layers. Your hypertable is
+    spread across the layers, so queries and `JOIN`s work and fetch the same
+    data as usual. Reading data from object storage adds only a few tens of
+    milliseconds in latency, which goes away for larger scans.
+
+## Architecture
+
+Data tiering works by periodically and asynchronously moving older chunks to S3
+storage. There, it's stored in a compressed columnar format well-suited for S3,
+namely, the Apache Parquet format. Data remains accessible both during and after
```
Comment (Loquacity) [inline cloud/data-tiering/index.md:48]:
```suggestion
storage. There, it's stored in the Apache Parquet format, which is a compressed 
columnar format well-suited for S3. Data remains accessible both during and after
```

File: cloud/data-tiering/index.md [hunk 4]
```diff
@@ -35,11 +36,40 @@ With data tiering, you get:
...
+migration.
+
+When you run your regular SQL queries, a behind-the-scenes process transparently
```
Comment (Loquacity) [inline cloud/data-tiering/index.md:51]:
```suggestion
When you run regular SQL queries, a behind-the-scenes process transparently
```

**Comment by Loquacity** [comment]
Is there a reason this is still in draft? Can we go ahead and merge?

**Comment by charislam** [comment]
No, Alaap wants edits, but data tiering is in transitional state between two different beta stages, so last I heard he's still thinking through the docs plan

**Comment by charislam** [comment]
Closing as requirements have changed, see https://github.com/timescale/docs-private/issues/84

**Pull request closed without merging**